t/Util.pm

```diff
@@ -890,4 +892,11 @@ EOT
     return join("\n", ($resp_out, $resp_err));
 }
 
+my $Test;
+sub test_is_passing {
+    $Test = Test::Builder->new
+        unless defined $Test;
+    $Test->is_passing;
```

Comment:
I think `Test::More->builder` returns the instance of Test::Builder and we can use it.

Reply:
Good point. Applied in 1f710a1.